ansible/roles/postgresql_server/tasks/main.yml

```diff
@@ -43,9 +43,20 @@
     state: 'directory'
     owner: '{{ postgresql_server__user }}'
     group: '{{ postgresql_server__group }}'
-    mode: '0755'
+    mode: '0750'
   when: postgresql_server__data_directory != '/var/lib/postgresql'
 
+
+- name: Make sure that custom PostgreSQL Log directory exists
+  file:
+    path: '{{ postgresql_server__log_directory }}'
+    state: 'directory'
+    owner: '{{ postgresql_server__user }}'
+    group: '{{ postgresql_server__group }}'
+    mode: '0750'
+  when: postgresql_server__log_directory != '/var/lib/postgresql'
```

Comment:
The role does not have `postgresql_server__log_directory` variable defined by default. In the generated `postgresql.conf` file the `log_directory` variable has the value `pg_log`. The default logs are stored in the `/var/log/postgresql/` directory, per cluster I think. Any idea what value should be the default? `/var/lib/postgresql` seems to be incorrect.

Reply:
from the pg_createcluster manpages: http://manpages.ubuntu.com/manpages/trusty/man8/pg_createcluster.8.html
```
  -l path, --logfile=path
           Explicitly set the path for the postmaster(1) server log file. Defaults to
           /var/log/postgresql/postgresql-version-cluster.log.
```

Reply:
Thus when you do specify it for pg_createcluster, you actually have to specify with the full path, ie. `/directory/filename.log`, though that gets "overriden" by the `postgres.conf` log file setting(s), and then becomes the value that `pg_lsclusters` reports (afaik), though there is a systemd time that will use that `/directory/filename.log' for the initial log file(s) before the postmaster started to read the `postgres.conf` file.

Reply:
I'm sorry that it takes so long to merge this, but something else bugs me. If you already create the directory by setting it as `item.log_directory` below, and you need to specify an absolute path, why additionally create some kind of global log directory? I think that the above task is not really needed. Thoughts?